pyproject.toml

```diff
@@ -9,4 +9,4 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 79
-target-version = ["py27"]
+target-version = ["py36"]
```

Comment:
This should be 3.5.

Reply:
I made it 36 on the assumption we'll drop 35 soon, but it's an easy change so sure 😄